test/e2e-test-application/cypress/e2e/tests/1-angular/luigi-client-link-manager-features-6.cy.js

```diff
@@ -0,0 +1,102 @@
+import { wrap } from "module";
+
+
+describe('Luigi Client linkManager Modal', () => {
+  beforeEach(() => {
+    cy.visitLoggedIn('/');
+  });
+
+    it('print stuff out so, I can check what functions are available etc.', () => {
+      cy.window().then(win => {
+        cy.log(win)
+        cy.log("----win.Luigi----")
+        cy.log(win.Luigi)
+        cy.log("----win.Luigi.navigation()----")
+        cy.log(win.Luigi.navigation())
+        cy.log("----win.Luigi.navigation().openAsModal() without cy.wrap----")
+        cy.log(win.Luigi.navigation().openAsModal('/projects/pr2'))
```

Comment:
I think this task for now should focus on LuigiClient rather than Luigi Core.
win.Luigi seems like an attempt at accessing Luigi Core API.
There are two mirrored functions on both Core and Client:
1 - [openAsModal on Core](https://docs.luigi-project.io/docs/luigi-core-api/?section=examples-28)
2 - [openAsModal on Client](https://docs.luigi-project.io/docs/luigi-client-api/?section=examples-30)

Here you focus on 2 instead.